src/components/Billing/ChargeItem/ChargeItemDefinitionPopover.tsx

```
@@ -0,0 +1,80 @@
+  const totalPrice = calculateTotalPrice(priceComponents);
+
+  return (
+    <Popover>
+      <PopoverTrigger asChild>
+        <button
+          className={cn(
+            "text-sm font-medium underline decoration-dotted underline-offset-4 cursor-pointer hover:text-primary-700 transition-colors",
+            className,
+          )}
+          aria-label={t("view_details")}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)